jq_service_parser.go

```diff
@@ -26,7 +26,7 @@ func NewJQServiceParser(cfg ServiceRegistrationConfig) *JQServiceParser {
 		product:      NewJQFieldParser(cfg.Product),
 		language:     NewJQFieldParser(cfg.Language),
 		framework:    NewJQFieldParser(cfg.Framework),
-		system:       NewJQFieldParser(""), //cfg.System),
+		system:       NewJQFieldParser(""), // cfg.System),
```

Comment:
this is a bug - can you fix this

```suggestion
		system:       NewJQFieldParser(cfg.System),
```